src/index.js

```diff
@@ -28,8 +28,10 @@ const history = syncHistoryWithStore(browserHistory, store);
 const ReduxApp = () => (
   <Provider store={store}>
     <Router history={history}>
-      <Route path="/" component={App}/>
-      <Route path="/foo" component={Foo}/>
+      <Route path="/" component={App}>
```

Comment:
@auredufour @turquoisemelon if you expand the code for this page and look at line 26, browserHistory is synced with store to make the history variable. It's from the code here I think we were shown in class >> https://github.com/reactjs/react-router-redux. Thoughts?